DEVREADME.md

```diff
@@ -74,6 +74,14 @@ docker exec indy-pool add-did-from-seed 000000000000000000000000Trustee9 TRUSTEE
 # docker exec indy-pool add-did "NkGXDEPgpFGjQKMYmz6SyF" "CrSA1WbYYWLJoHm16Xw1VEeWxFvXtWjtsfEzMsjB5vDT"
 ```
 
+### Setup Cheqd Ledger
+
+In addition, there's also a docker command to run a cheqd test network.
```

Comment:
Would be good to also have an advice about skipping cheqd tests like we did for postgres